### ISSUE TITLE
Log errors from DOMDocument::loadHTML() using libxml_get_errors()

### ISSUE THREAD

**Issue by shweshi** [shweshi/OpenGraph#64]
Retrieve the error by using libxml_get_errors() perform logging or return the list of issues to the caller.

```
$html = $this->curl_get_contents($url, $lang);
        /**
         * parsing starts here:.
         */
        $doc = new DOMDocument();

        $libxml_previous_state = libxml_use_internal_errors(true);
        $doc->loadHTML('<?xml encoding="utf-8" ?>'.$html);

        //catch possible errors due to empty or malformed HTML
        Log::warning(libxml_get_errors())

        libxml_clear_errors();
        // restore previous state
        libxml_use_internal_errors($libxml_previous_state);
```

Using Log::warning(libxml_get_errors()) is not compatible with PHP 5.6

**Comment by alinmiron** [comment]
libxml_get_errors() is available in PHP 5 >= 5.1.0 and  PHP 7, according to PHP documentation.

Log::warning is available in illuminate/log package (  'warning'   => MonologLogger::WARNING) way before the version 5.0 of the package; see https://github.com/illuminate/log/blob/5.0/Writer.php ; also, the minimum supported PHP version would be >=5.4.0

we can use in composer "illuminate/log": "^5.0|^6.0|^7.0|^8.0", 
in this way I think that we support many framework versions.

**Comment by shweshi** [comment]
> libxml_get_errors() is available in PHP 5 >= 5.1.0 and PHP 7, according to PHP documentation.
> 
> Log::warning is available in illuminate/log package ( 'warning' => MonologLogger::WARNING) way before the version 5.0 of the package; see https://github.com/illuminate/log/blob/5.0/Writer.php ; also, the minimum supported PHP version would be >=5.4.0
> 
> we can use in composer.json "illuminate/log": "^5.0|^6.0|^7.0|^8.0",
> in this way I think that we support many framework versions and of course many PHP versions.
> 
> I would like to also use another param on the OpenGraph::fetch method to allow users to enable/disable the logging of the malformed HTML parsing.
> I'll come with a Pull Request soon

This looks good @alinmiron Thanks.

**Comment by shweshi** [comment]
Fixed in #65
